node/test/tasklib.ts

```diff
@@ -803,6 +803,20 @@ describe('Test vsts-task-lib', function () {
             assert.equal(tc.toString(), '##vso[some.cmd]a message');
             done();
         })
+        it('handles boolean properties', function (done) {
+            this.timeout(1000);
+
+            var tc = new tcm.TaskCommand('some.cmd', { foo: false }, 'a message');
```

Comment:
properties are a string, string dictionary as Eric noted above.  I'm confused why you're passing a boolean.  See TaskCommand.ts